config/settings/integration.yml

```diff
@@ -0,0 +1,7 @@
+---
+:log:
+  :level_rails: debug
+:webservices:
+  :url: http://localhost:4000
+:server:
+  :session_store: memory
```

Comment:
Is this setting working yet? If so, do we need the changes to session_store.rb?

Reply:
Yeah, sorry, I rebased this change into the first commit, but probably should have left it as a separate `[FIXUP]` commit for a bit.

This comes from something entirely different than the `config/initializer`, and is used for the `lib/token_store.rb` here:

https://github.com/ManageIQ/manageiq/blob/c58a461cd63e49a1d890e560486ac963bb2ec863/lib/token_store.rb#L6

I specifically added this section based on this commit:

https://github.com/ManageIQ/manageiq/commit/24422a34f33b6c7b10faf63320b23042ec3f6632#diff-ba53bed3794c5d72b451fcb901c2ab28

And I "think" this is where the other warning is coming from, but not sure.

* * *

I think for more debugging, I might try just running a server inline (without a `.spawn`/`.detach`) command via `Open3.popen2e` to see what is printed out for an error.  I assume there has to be something... right?